src/Sportradar.OddsFeed.SDK/Entities/REST/Internal/EntitiesImpl/Competitor.cs

```diff
@@ -266,6 +266,12 @@ public async Task<ICategorySummary> GetCategoryAsync()
         /// <value>The short name</value>
         public string ShortName => GetOrLoadCompetitor()?.ShortName;
 
+        public async Task EnsureProfileLoaded()
+        {
+            // Ensure cache is loaded
+            await GetOrLoadCompetitorAsync();
```

Comment:
This discards the result which is unfortunate but felt like the best way to make sure the Competitor is loaded into the cache